src/createServer.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { router: usersRouter } = require('./routes/users');
+const { router: expenseRouter } = require('./routes/expenses');
+const { initialize: inizializeUsers } = require('./services/users');
+const { initialize: inizializeExpenses } = require('./services/expenses');
+
+const app = express();
+
+app.use('/users', express.json(), usersRouter);
+app.use('/expenses', express.json(), expenseRouter);
```

Comment:
Instead of adding `express.json()` to every route, add it above using `app.use()`
```suggestion
app.use(express.json());

app.use('/users', usersRouter);
app.use('/expenses', expenseRouter);
```